.eslintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": ["react", "prettier"],
+  "parser": "babel-eslint",
+  "env": {
+    "node": true,
+    "es6": true,
+    "mocha": true,
+    "browser": true,
+    "commonjs": true,
+    "jest": true
+  },
+  "rules": {
+    "new-cap": 0,
+    "import/prefer-default-export": "off",
+    "prettier/prettier": "error",
+    "import/no-unresolved": 0,
+    "no-plusplus": 0,
+    "react/sort-comp": 0,
+    "react/prop-types": 0,
+    "no-unused-expressions": 0,
```

Comment:
Though `Create React App` might be doing tree shaking for us behind the scenes. I would suggest we turn on this rule. It will help us prevent having dead code especially variables and imports (code that isn't used anywhere) in our codebase.